modules/bim/spec/factories/bcf_issue_factory.rb

```diff
@@ -111,7 +111,7 @@
     end
     stage { nil }
     labels { [] }
-    sequence(:index) { |n| n }
+    sequence(:index)
```

Comment:
the identity function is the default argument to `sequence`, no need to override it